src/main/java/net/minecraftforge/common/util/TeleporterHelper.java

```diff
@@ -0,0 +1,36 @@
+package net.minecraftforge.common.util;
+
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.MathHelper;
+import net.minecraft.world.DimensionType;
+import net.minecraft.world.World;
+import net.minecraft.world.border.WorldBorder;
+
+public class TeleporterHelper
```

Comment:
There is no reason for this stuff to exist anymore, at least not as public forge API, just put the methods in your test mod if they are still needed by it.

Reply:
Fair enough. I wasn't sure if I should've kept it or not

Reply:
Ya remove it, it serves no purpose